README.md

```diff
@@ -65,7 +70,7 @@ or create a issue in this repo. We will be more than glad to answer.
 
 Support us with a monthly donation and help us continue our activities.
 
-[![Backers](https://opencollective.com/spyder/backers.svg)](https://opencollective.com/spyder#support)
+[![Backers](https://opencollective.com/spyder/backers.svg?width=1000)](https://opencollective.com/spyder#support)
```

Comment:
Could you change this section to be like the [last one](https://github.com/spyder-ide/spyder-kernels#sponsors) of spyder-kernels' Readme? We're trying to show the same section in all our projects' Readmes.

Reply:
Done